code/datums/diseases/chronic_illness.dm

```diff
@@ -16,6 +16,7 @@
 	bypasses_immunity = TRUE//Bubber addition
 	severity = DISEASE_SEVERITY_UNCURABLE
 	bypasses_immunity = TRUE
+	var/being_stealthy = TRUE // Bubber addition
```

Comment:
Why are you uh, adding a new bolean var but it doesnt seem to do anything. You are still checking for exact stage of the disease

Reply:
Stage is being checked for timing. This variable is updated to false if a visible symptom occurs during stages two or three. If it is still true when the stage rolls over to four, then no visible symptoms have occurred yet by stage four, and a visible symptom is forced to occur.